cfme/tests/control/test_actions.py

```diff
@@ -15,13 +15,19 @@
 # TODO: Move the SOAP calls to UI checks since SOAP is deprecated
 import fauxfactory
 import pytest
+import cfme.fixtures.pytest_selenium as sel
```

Comment:
Inside the test run, the pytest_selenium is available as `sel` inside the pytest namespace.

so you can write `pytest.sel.click()` and so.